workflows/prognostic_c48_run/runtime/factories.py

```diff
@@ -24,3 +27,37 @@ def get_emulator_adapter(
             diagnostic_variables=set(config.diagnostic_variables),
             timestep=timestep,
         )
+
+
+def get_overrider_adapter(
```

Comment:
You should make this private since it is confusing to have two "public" factories for OverriderAdpaters. One which returns just overrider_adapters and a more general one `get_fv3_physics_adapter` which can also return emulators.

Reply:
I deleted the `get_fv3_physics_adapter` function.